src/test/java/com/google/api/codegen/grpcmetadatagen/testdata/library_pkg.yaml

```diff
@@ -28,18 +30,24 @@ gax_version:
     upper: '0.16.0'
   nodejs:
     lower: '^0.7.0'
+  java:
+    lower: '0.6.1'
```

Comment:
Not yet, but I assume we will be using it to generate other metadata. I can remove it for now.